web/src/tippyjs.js

```diff
@@ -344,6 +344,23 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    delegate("body", {
+        target: [".user_row .actions button"],
+        trigger: "mouseenter",
+        onShow(instance) {
+            if ($(instance.reference).hasClass("deactivate")) {
+                instance.setContent($t({defaultMessage: "Deactivate"}));
+                return true;
+            } else if ($(instance.reference).hasClass("reactivate")) {
+                instance.setContent($t({defaultMessage: "Reactivate"}));
+                return true;
+            }
+            return false;
```

Comment:
This function will be triggered for the following buttons:-

- `Edit User` button
- `Deactivate User` button
- `Reactiavte User` button

When the function is triggered due to `Edit User` button we return `False` so that no tooltip is shown, otherwise return `True` to show the tooltip